kgforge/specializations/resolvers/ontology_resolver.py

```diff
@@ -39,12 +39,15 @@ def mapper(self) -> Callable:
         return DictionaryMapper
 
     def _resolve(self, text: Union[str, List[str]], target: Optional[str], type: Optional[str],
-                 strategy: ResolvingStrategy, resolving_context: Any, limit: Optional[str], threshold=Optional[float]) \
+                 strategy: ResolvingStrategy, resolving_context: Any, limit: Optional[int], threshold=Optional[float]) \
```

Comment:
The limit: Optional[int] should also be done in all [resolvers](https://github.com/BlueBrain/nexus-forge/tree/f1bdac4a737b6936b5fd3425222c3db0459de206/kgforge/specializations/resolvers)